stubs/aws-xray-sdk/aws_xray_sdk/core/async_context.pyi

```diff
@@ -1,15 +1,19 @@
 from _typeshed import Incomplete
+from asyncio.events import AbstractEventLoop
+from typing_extensions import ParamSpecArgs, ParamSpecKwargs
 
 from .context import Context as _Context
 
 class AsyncContext(_Context):
-    def __init__(self, *args, loop: Incomplete | None = None, use_task_factory: bool = True, **kwargs) -> None: ...
+    def __init__(
+        self, *args: ParamSpecArgs, loop: Incomplete | None = None, use_task_factory: bool = True, **kwargs: ParamSpecKwargs
+    ) -> None: ...
```

Comment:
I've never seen `ParamSpecArgs` used like this. It seems that `*args` and `**kwargs` are just passed on to the super class in the implementation. I suggest to just copy the super class argument:

```suggestion
    def __init__(
        self, context_missing: str = "LOG_ERROR", loop: Incomplete | None = None, use_task_factory: bool = True
    ) -> None: ...
```